taskvine/src/manager/vine_manager.c

```diff
@@ -269,10 +269,8 @@ static void handle_worker_timeout(struct vine_manager *q, struct vine_worker_inf
 	// debug(D_VINE, "Handling timeout request");
 	HASH_TABLE_ITERATE(w->current_files, cachename, replica)
 	{
-		//	debug(D_VINE, "Looking at file %s", cachename);
 		if (strncmp(cachename, "temp-rnd-", 9) == 0) {
```

Comment:
While you are at it, please change this to consult `replica->type` instead of dinking around with the cachename.